EmbeddedSocial/Sources/Model/Services/Comments/CommentsService.swift

```diff
@@ -218,8 +217,9 @@ class CommentsService: BaseService, CommentServiceProtocol {
                     resultHandler(command.comment)
                 } else if self.errorHandler.canHandle(error) {
                     self.errorHandler.handle(error)
+                } else {
```

Comment:
In this case, those who are subscribed to the Failure can not notify the controller(through the whole chain)  to unlock the UI or make any other manipulations. How would you suggest to solve this situation?